src/main/java/com/amihaiemil/docker/Containers.java

```diff
@@ -75,10 +76,25 @@ Container create(
     Container create(final JsonObject container) throws IOException;
     
     /**
-     * Return all the Containers, not only the running ones.
+     * Return all Containers, not only the running ones.
      * @return Iterator over all the containers.
      */
     Iterator<Container> all();
+
+    /**
+     * Whether to request the size of containers (fields SizeRw and SizeRootFs).
+     * @param withSize Return the size of containers (SizeRw and SizeRootFs).
+     * @return Containers with modified size flag.
+     */
+    Containers withSize(final boolean withSize);
```

Comment:
@milux I think that this could be implemented as a decorator to `Containers`, which returns decorated `Container` objects with size information

Reply:
Same as above, maybe another interface?

Reply:
And here is the decorator once again... see above. :wink: